content/install-user-css.js

```diff
@@ -107,6 +78,51 @@ function initInstallPage({style, dup}, sourceLoader) {
     if (location.protocol === 'file:') {
       initLiveReload(sourceLoader);
     }
+
+    function buildPage() {
+      return $element({className: 'container', appendChild: [
+        $element({className: 'header', appendChild: [
+          $element({tag: 'h1', appendChild: [
+            style.name,
+            $element({tag: 'small', className: 'meta-version', appendChild: style.version})
+          ]}),
+          $element({tag: 'p', appendChild: style.description}),
```

Comment:
Any particular reason you didn't use `textContent:` ? I think it'd make the structure more obvious.

Reply:
I just never thought that. I'll change them later.